config/hsm.py

```diff
@@ -0,0 +1,8 @@
+# Enable the basic HSM shapes, which is preferred over SdssShape.
+# `config` is a SourceMeasurementConfig.
+
+import lsst.meas.extensions.shapeHSM
+
+config.plugins.names |= ["ext_shapeHSM_HsmSourceMoments", "ext_shapeHSM_HsmPsfMoments"]
+config.slots.shape = "ext_shapeHSM_HsmSourceMoments"
+config.slots.psfShape = "ext_shapeHSM_HsmPsfMoments"
```

Comment:
I assume you meant for this file to be shared?

Reply:
Yes, it's intentionally shared between them. Eventually I think this config will end up in {{characterizeImageConfig.setDefaults}} via [DM-34969](https://jira.lsstcorp.org/browse/DM-34969), but this will do for us for now.